app/src/screens/Splash.tsx

```diff
@@ -304,7 +321,7 @@ const Splash: React.FC = () => {
         setStep(5)
         const options = {
           config: {
-            label: 'BC Wallet',
```

Comment:
Would it be possible for the user to enter a blank name and break the wallet install? Since our poking around the other day it needs to be file-system safe characters. Probably not names like `../../my-wallet`.